### PR TITLE
gmt 5.2.1

### PR DESCRIPTION
**Pull request by claudiodsf**

### CODE REVIEW

**Comment by claudiodsf** [comment]
Not sure why the test failed, but it seems unrelated.

File: gmt.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Gmt < Formula
   homepage "http://gmt.soest.hawaii.edu/"
-  url "ftp://ftp.soest.hawaii.edu/gmt/gmt-5.1.2-src.tar.bz2"
-  mirror "ftp://ftp.star.nesdis.noaa.gov/pub/sod/lsa/gmt/gmt-5.1.2-src.tar.bz2"
-  mirror "ftp://gd.tuwien.ac.at/pub/gmt/gmt-5.1.2-src.tar.bz2"
-  sha1 "aed2d79f83717c158adae1f634e6c3d8e5a09515"
+  url "ftp://ftp.soest.hawaii.edu/gmt/gmt-5.2.1-src.tar.xz"
+  mirror "ftp://ftp.star.nesdis.noaa.gov/pub/sod/lsa/gmt/gmt-5.1.2-src.tar.xz"
```
Comment (dpo) [inline gmt.rb]: Shouldn't this refer to version `5.2.1`?

Comment (claudiodsf) [inline gmt.rb]: Sure, thanks. I'll fix it right away.

**Comment by claudiodsf** [comment]
I'm not sure why tests are failing on Travis, but my bet is that it is due to the recent `hdf5` update (https://github.com/Homebrew/homebrew-science/pull/3027) affecting `netcdf` (I had to recompile `netcdf` on my machine to make `gmt` compile again).

As from https://github.com/Homebrew/homebrew-science/pull/2749, `hdf5` updates should not be pulled without bumping the revision of all the deps.

**Comment by claudiodsf** [comment]
Checked the logs 9http://bot.brew.sh/job/Homebrew%20Science%20Pull%20Requests/3634/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/install_homebrew_science_gmt/), that's indeed the case.

The solution is to recompile `netcdf`, i.e. bump its revision number.

**Comment by dpo** [comment]
@BrewTestBot test this please

**Comment by claudiodsf** [comment]
@dpo Strange that it's failing again. Looks like the latest revision of `netcdf` was not yet on Travis.
Let's try testing it again.

**Comment by claudiodsf** [comment]
@BrewTestBot test this please

**Comment by claudiodsf** [comment]
Ah ok, if I well understood, only Jenkins is triggered by BrewTestBot, not Travis. Anyway, it seems that the Jenkins build is fine, so maybe this can be merged...

**Pull request closed without merging**